tests/containers/rootless_podman.pm

```diff
@@ -104,28 +104,11 @@ sub run {
     # By default the storage driver is set to btrfs if /var is in btrfs
     # but if the home partition is not btrfs podman commands will fail with
     # Error: "/home/bernhard/.local/share/containers/storage/btrfs" is not on a btrfs filesystem
-    if (script_output("podman info 2>&1", proceed_on_failure => 1) =~ m/prerequisites for driver not satisfied/) {
```

Comment:
FYI this workaround was removed as https://bugzilla.suse.com/show_bug.cgi?id=1197093 has been resolved